PYTHIA6/AliPythia6/AliGenPythiaPlus.cxx

```diff
@@ -43,10 +32,12 @@
 #include "AliStack.h"
 #include "AliRunLoader.h"
 #include "AliMC.h"
+#include "AliLog.h"
 #include "PyquenCommon.h"
 
-ClassImp(AliGenPythiaPlus)
-
+/// \cond CLASSIMP
+ClassImp(AliGenPythiaPlus);
+/// \endcond
 
```

Comment:
This is needed for doxygen, I can remove it.

There are only a couple of reformating, most of the changes are replacing one old piece of code by another.